src/features/user-profile/components/UpdateProfilePictureForm.tsx

```diff
@@ -152,9 +162,6 @@ export const UpdateProfilePictureForm: FC<Props> = ({ user, onSubmit, defaultVal
           )}
           <UserProfilePicture user={user} size='sm' radius={64} />
         </UserProfilePictureContainer>
-        <Form.Control.Feedback type='invalid' role='alert'>
```

Comment:
Yeah. I agree.

The showErrorMessage message just causes a toast to be shown to the user. Since we don't test the toast messages in other cases, I'll go ahead and remove the test that is failing.

Reply:
This would be the perfect time for an integration test and/or something like Selenium since the test would be around the functionality for the user rather than a low value unit test.